src/09-word-problems.js

```diff
@@ -80,7 +93,15 @@ function getCartTotal(products) {}
     compareLocations(address1, address2);
     //> "Same city."
  */
-function compareLocations(address1, address2) {}
+function compareLocations(address1, address2) {
+   if (address1.state === address2.state && address1.city !== address2.city) {
+     return "Same state."
```

Comment:
Lets break apart these long chains of logic on their comparison operator
 every ` && ` or ` || ` should be the last item on a line and variable moved one down.